### PR TITLE
add @adobe/react-native-aepcore

### PR DESCRIPTION
**Pull request by blumk**
# 📝 Why & how
Adds a new library `@adobe/react-native-aepcore`, see https://www.npmjs.com/package/@adobe/react-native-aepcore

# ✅ Checklist
- [x] Added library to **`react-native-libraries.json`**
- [ ] Updated library in **`react-native-libraries.json`**

### CODE REVIEW

**Comment by blumk** [comment]
@Simek validation keeps failing, npm package name differs from github url (monorepo)

File: react-native-libraries.json [hunk 1]
```diff
@@ -12124,5 +12124,24 @@
     "expoGo": true,
     "dev": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/adobe/aepsdk-react-native",
```
Comment (Simek) [inline react-native-libraries.json:12129]: Hey @blumk, this fail is correct. In monorepos `githubUrl` needs to point to the directory, where given package resides:

```suggestion
    "githubUrl": "https://github.com/adobe/aepsdk-react-native/tree/main/packages/core",
```

File: react-native-libraries.json [hunk 2]
```diff
@@ -12124,5 +12124,24 @@
...
+    "web": false,
+    "expoGo": true,
+    "windows": false,
+    "macos": false,
+    "tvos": false,
+    "visionos": false,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
+    "newArchitecture": false
```
Comment (Simek) [inline react-native-libraries.json:12145]:
```suggestion
    "android": true,
    "expoGo": true
```

Formatting nit, `false` flags are redundant.

Comment (blumk) [reply react-native-libraries.json]: updated - thanks! 🙌

**Pull request merged**